zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.4.5
+
+### Bug fixes
+
+- NVM backups can now be restored onto 800 series controllers

```

Comment:
I think we should prefix the fixes and feature with the context of the change, ie `Z-Wave JS:` if it was done in the driver, `Add-on:` if it was done in the add-on, etc.

We did this some time ago but stopped doing it for some reason. It's hard to read the changelog without that and know when we changed the add-on and when we just bumped the driver or server.

Reply:
agreed, I seem to have forgotten to do it properly. Hopefully I caught everything